core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/AttributeComponent.scala

```diff
@@ -33,9 +33,7 @@ trait AttributeRecord[OWNER_ID] {
   val deletedDate: Option[Timestamp]
 }
 
-trait AttributeScratchRecord[OWNER_ID] extends AttributeRecord[OWNER_ID] {
-  val transactionId: String
-}
+trait AttributeScratchRecord[OWNER_ID] extends AttributeRecord[OWNER_ID]
```

Comment:
the transactionId is a leftover from when we had a persistent `ENTITY_ATTRIBUTE_SCRATCH` table which was used for multiple transactions. In that setup, different rows had different transactionIds.

Back in 2021 we switched from a persistent table to temp tables, but we retained the transactionId column. However, the transactionId value is the same for every row in the temp table, so it is completely extraneous.